docs/source/publish_and_share_kedro_viz_on_aws.md

```diff
@@ -0,0 +1,119 @@
+# Publish and share via AWS
+
+This page describes how to publish Kedro-Viz to AWS to share it with others. It uses the spaceflights tutorial as an example.
+
+## Setup your kedro project 
+
+If you haven't installed Kedro {doc}`follow the documentation to get set up<kedro:get_started/install>`. 
+
+```{important}
+We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.19.1). To check the version installed, type `kedro -V` in your terminal window.
+```
+
+In your terminal window, navigate to the folder you want to store the project. Generate the spaceflights tutorial project with all the code in place by using the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas):
+
+
+```bash
+kedro new --starter=spaceflights-pandas
+```
+
+When prompted for a project name, you can enter anything, but we will assume `Spaceflights` throughout.
+
+When your project is ready, navigate to the root directory of the project. Install the dependencies from the project root directory by typing the following in your terminal:
+
+```bash
+pip install -r requirements.txt
+```
+
+Kedro-Viz requires specific minimum versions of `fsspec[s3]`, and `kedro` to publish your project. Ensure you have these correct versions by updating the `requirements.txt` file of the Kedro project to add the following:
```

Comment:
Since we separated s3fs dependency from 8.0.0, we only need minimum versions of fsspec, kedro and not fsspec[s3]